jenkins-scripts/lib/_homebrew_github_commit.bash

```diff
@@ -8,54 +8,51 @@
 PR_URL_export_file=${PR_URL_export_file:-${WORKSPACE}/pull_request_created.properties}
 
 echo '# BEGIN SECTION: check variables'
+if [ -z "${COMMIT_MESSAGE}" ]; then
+  if [ -z "${SKIP_COMMIT}" ]; then
+    echo One of COMMIT_MESSAGE or SKIP_COMMIT must be specified
+    exit -1
+  fi
+fi
 if [ -z "${PULL_REQUEST_BRANCH}" ]; then
   echo PULL_REQUEST_BRANCH not specified
   exit -1
 fi
+if [ -z "${PULL_REQUEST_TITLE}" ]; then
+  if [ -z "${PULL_REQUEST_URL}" ]; then
+    echo One of PULL_REQUEST_TITLE or PULL_REQUEST_URL must be specified
+    exit -1
+  fi
+fi
 if [ -z "${TAP_PREFIX}" ]; then
   echo TAP_PREFIX not specified
   exit -1
 fi
-# PACKAGE_ALIAS and VERSION are required if a pull request doesn't yet exist
-if [ -z "${PULL_REQUEST_URL}" ]; then
-  if [ -z "${PACKAGE_ALIAS}" ]; then
-    echo PACKAGE_ALIAS not specified
-    exit -1
-  fi
-  if [ -z "${VERSION}" ]; then
-    echo VERSION not specified
-    exit -1
-  fi
-fi
 echo '# END SECTION'
 
 GIT="git -C ${TAP_PREFIX}"
 
-DIFF_LENGTH=`${GIT} diff | wc -l`
-if [ ${DIFF_LENGTH} -eq 0 ]; then
-  echo No formula modifications found, aborting
-  exit -1
-fi
-echo ==========================================================
-${GIT} diff
-echo ==========================================================
-echo '# END SECTION'
-
-echo
 echo '# BEGIN SECTION: commit and pull request creation'
-${GIT} config user.name "OSRF Build Bot"
-${GIT} config user.email "osrfbuild@osrfoundation.org"
+echo
 ${GIT} remote -v
 # check if branch already exists
 if ${GIT} rev-parse --verify ${PULL_REQUEST_BRANCH} ; then
   ${GIT} checkout ${PULL_REQUEST_BRANCH}
 else
   ${GIT} checkout -b ${PULL_REQUEST_BRANCH}
 fi
-if [ -n "${PACKAGE_ALIAS}" ]; then
-  COMMIT_MESSAGE_PREFIX="${PACKAGE_ALIAS}: "
+if [ -z "${SKIP_COMMIT}" ]; then
```

Comment:
Asumming that we can store true|false 1|0 in the SKIP_COMMIT the check is only working for empty values and probably do something unexpected if we ever set `SKIP_COMMIT=true`

Reply:
https://github.com/gazebo-tooling/release-tools/pull/1390/commits/2690801dea0dbdb997962f7ebd2c3edaf64fe25c